config/config.go

```diff
@@ -71,6 +71,12 @@ func (cfg *config) init(env *AppConfig) error {
 		if err != nil {
 			return err
 		}
+	} else {
+		// If no LNDCertFile is provided, we create a new one during startup
```

Comment:
what do you mean by "create a new one"? this comment seems incorrect to me

Reply:
This is a bit ungraceful - now all users even ones who do not run LND will have this entry in their user config

Reply:
should we only run that code if `cfg.Env.LNBackendType` is LND?

Reply:
I think the comment is misleading, it just skips the tls verification if no certificate is provided. And for that to happen, we need to leave this empty. So other backends should be fine.